diff --git a/node_modules/expo-updates/.bun-tag-b0a0b6296a1a348d b/.bun-tag-b0a0b6296a1a348d
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/ios/EXUpdates/AppLoader/AppLoader.swift b/ios/EXUpdates/AppLoader/AppLoader.swift
index 19a52362232a4589a91c1023d8b46dffbdb4239f..7b804750b9e49a8c954a4f67d844411db96d3ea3 100644
--- a/ios/EXUpdates/AppLoader/AppLoader.swift
+++ b/ios/EXUpdates/AppLoader/AppLoader.swift
@@ -344,7 +344,9 @@ open class AppLoader: NSObject {
             }
           }
           // This replaces the old force try
-          assert(contents != nil)
+          if !UpdatesUtils.isNativeDebuggingEnabled() {
+            assert(contents != nil)
+          }
           if let contents = contents {
             existingAsset.contentHash = UpdatesUtils.hexEncodedSHA256WithData(contents)
             existingAsset.downloadTime = Date()
