diff --git a/scripts/ios/xcode_env_generator.rb b/scripts/ios/xcode_env_generator.rb
index b7d5f88913d4f14a272d97b50cba03f35ac06d01..a0774642680e819d447104e95e900eb79f76054d 100644
--- a/scripts/ios/xcode_env_generator.rb
+++ b/scripts/ios/xcode_env_generator.rb
@@ -16,6 +16,7 @@ def generate_or_remove_xcode_env_updates_file!()
       export FORCE_BUNDLING=1
       unset SKIP_BUNDLING
       export RCT_NO_LAUNCH_PACKAGER=1
+      export NODE_ENV=production
 EOS
     )
   else
