diff --git a/scripts/react-native-xcode.sh b/scripts/react-native-xcode.sh
index b5d50ccb4bb4967edb575995ba727acce99dfd4a..c25861c5120539f9827af612430aea61e0300be6 100755
--- a/scripts/react-native-xcode.sh
+++ b/scripts/react-native-xcode.sh
@@ -45,7 +45,7 @@ case "$CONFIGURATION" in
       echo "Bundling for physical device. Use the SKIP_BUNDLING flag to change this behavior."
     fi
 
-    DEV=true
+    DEV=false
     ;;
   "")
     echo "$0 must be invoked by Xcode"
